backend/src/main/kotlin/com/infowings/catalog/data/objekt/ObjectService.kt

```diff
@@ -199,14 +205,17 @@ class ObjectService(
             )
         }
 
-        return PropertyCreateResponse(
-            propertyCreateResult.id,
-            Reference(propertyCreateResult.objectId, propertyCreateResult.objectVersion),
-            Reference(propertyCreateResult.rootValueId, propertyCreateResult.rootValueVersion),
-            propertyCreateResult.name,
-            propertyCreateResult.description,
-            propertyCreateResult.version
-        )
+        return transaction(db) {
```

Comment:
Why wrapping the response construction inside the transaction? It is intended that transaction will return all the necessary data in the "Result" wrapper and the response construction will be allowed without the transaction.